### PR TITLE
ARCH-1919 - Add env var

### PR DESCRIPTION
**Pull request by danielle-casella-adams**
# Summary of PR changes

This should reduce extra steps when I want to use the tagged version and local version in the reusable workflows.  

## PR Requirements
- [x] For major, minor, or breaking changes, at least one of the commit messages contains the appropriate `+semver:` keywords.
  - See the *Incrementing the Version* section of the repository's README.md for more details.
- [x] The action code does not contain sensitive information.

*NOTE: If the repo's workflow could not automatically update the `README.md`, it should be updated manually with the next version.  For javascript actions, if the repo's workflow could not automatically recompile the action it should also be updated manually as part of the PR.*

### CODE REVIEW

**Review by JosephDSchwartz — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -49,14 +49,14 @@ jobs:
...
-        uses: im-open/did-custom-action-code-change@v1.0.2
+        uses: im-open/did-custom-action-code-change@v1.0.3
         with:
           files-with-code: 'action.yml,package.json,package-lock.json'
           folders-with-code: 'src,dist'
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Commit unstaged changes if there are code changes
-        if: steps.action-code.outputs.HAS_CHANGES == 'true'
+        if: steps.action-code.outputs.HAS_CHANGES == 'true' # can also use env.HAS_CHANGES
```
Comment (JosephDSchwartz) [inline README.md]: Are we worried at all that `HAS_CHANGES` seems like a potentially common variable name and might conflict with workflows this is used in?

Comment (danielle-casella-adams) [reply README.md]: I was literally just going to fix this. :)  In my other action where I made a similar PR I changed it to FILE_HAS_CHANGED.  So I'll do something similar here.

Comment (danielle-casella-adams) [reply README.md]: This one has bee renamed to CODE_HAS_CHANGED.  I don't like that the output and env don't match but changing the output isn't realistic at this point.

Comment (JosephDSchwartz) [reply README.md]: We could update the output and make a major version update 😃 But that might be too much. I just 100% agree that having the output and env names be different sucks.

**Review by JosephDSchwartz — APPROVED** [review]

**Pull request merged**